packages/calypso-e2e/src/lib/pages/signup/signup-pick-plan-page.ts

```diff
@@ -28,18 +28,27 @@ export class SignupPickPlanPage {
 	 * @returns {Promise<SiteDetails>} Details of the newly created site.
 	 */
 	async selectPlan( name: Plans ): Promise< NewSiteResponse > {
+		await Promise.all( [
+			this.page.waitForURL( /.*start\/plans.*/ ),
+			this.page.waitForLoadState(),
```

Comment:
**comment** - I'm a little iffy on this one, because Playwright _should_ be waiting for the `load` event to fire on page load **and** perform actionability checks when performing actions (further down, in `selectPlan`). 

But clearly the flakiness is due to Playwright clicking on the button before the page is ready to accept such actions.